packages/E2ETest/wdio/test/DirectManipulation.spec.ts

```diff
@@ -5,28 +5,27 @@
 
 import HomePage from '../pages/HomePage';
 import DirectManipulationPage from '../pages/DirectManipulationPage';
-import assert from 'assert';
 import { DIRECT_MANIPULATION_TESTPAGE } from '@react-native-windows/tester/js/examples-win/LegacyTests/Consts';
 
 beforeAll(() => {
-  HomePage.goToTestPage(DIRECT_MANIPULATION_TESTPAGE);
+  HomePage.goToComponentExample(DIRECT_MANIPULATION_TESTPAGE);
 });
 
 describe('DirectManipulationTest', () => {
   it('measureInWindow Success', () => {
     const result = DirectManipulationPage.clickMeasureInWindowAndGetResult();
-    assert.ok(
```

Comment:
All of the specs were for some reason using Node's assertion API, which throws an exception, instead of Jasmine's which is integrated into the test runner. The community using Jest uses Jasmine-style matchers as well, so I moved everything over since we get better integration and more alignment with the community.